include/aspect/material_model/equation_of_state/multicomponent_compressible.h

```diff
@@ -34,13 +34,18 @@ namespace aspect
     {
       /**
        * A compressible equation of state that is intended for use with multiple compositional
-       * fields. For each material property, the user supplies a comma delimited list of
-       * length N+1, where N is the number of compositional fields used in the computation.
-       * The first entry corresponds to the "background" (which is also why there are N+1 entries).
+       * fields and potentially phases.
        *
-       * If a single value is given, then all the compositional fields are given
-       * that value. Other lengths of lists are not allowed. The material parameters
-       * for each compositional field are calculated self-consistently,
+       * For models without phase transitions, each material property is specified as a comma
+       * delimited list of length N+1 for the background and compositional fields (N) For models
+       * with phase transitions, the list needs to contain each field name, including the background,
+       * for a total of N+1 names, and for each of these names, specify the value for each phase
+       * Therefore, the total number of values given is N+P+1, with P = sum(P_c) the total number of
+       * phase transitions, summed over all phases. The format is background: value1|value2|...|valueP_1+1,
+       * field1:value1|...|valueP_2+1, ..., fieldN: value1|...|valueP_N+1. If only one value is given,
+       * then all fields/phases use the same value. Other lengths of lists are not allowed.
```

Comment:
I think this part is not needed here (since a user would not look for it in the .h file). 
```suggestion
       * phase transitions, summed over all phases. 
```